### PR TITLE
Create wildcard_ssl_overview

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 1339 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-e11e075a3/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1341 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-fac44fdbc/how-to/). valid for 14 days

**Comment by catlook** [comment]
@BadCrumpany final sign off, pls?

**Comment by rackernexus** [comment]
Build 1451 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-afcc077db/how-to/). valid for 14 days

File: content/cloud-servers/wildcard_ssl_overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+permalink: wildcard_ssl_overview/
+audit_date:
+title: Wildcard SSL overview
+type: article
+created_date: ‘2020-03-18’
+created_by: Benji Ivey
+last_modified_date: 
+last_modified_by:
```
Comment (stephamon) [inline content/cloud-servers/wildcard_ssl_overview.md]: Add audit_date, last_modified_date, and last_modified-by

File: content/cloud-servers/wildcard_ssl_overview.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+type: article
+created_date: ‘2020-03-18’
+created_by: Benji Ivey
+last_modified_date: 
+last_modified_by: 
+product: Cloud Servers
+product_url: cloud-servers
+---
+
+## What are wildcard SSL certificates?
```
Comment (stephamon) [inline content/cloud-servers/wildcard_ssl_overview.md]: I'd recommend deleting this header. We also don't use h2 level.

File: content/cloud-servers/wildcard_ssl_overview.md (hunk 1 above)
Comment (BadCrumpany) [reply content/cloud-servers/wildcard_ssl_overview.md]: Great, thank you!

**Comment by BadCrumpany** [comment]
@catlook Looks good to me!

**Comment by rackernexus** [comment]
Build 1459 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-38ab44b06/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1461 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-0be31a41e/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1466 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-0be31a41e/how-to/). valid for 14 days

**Pull request merged**